src/Lunr/Core/Tests/ConfigurationLoadEnvironmentTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * This file contains the ConfigurationLoadFileTest class.
+ *
+ * SPDX-FileCopyrightText: Copyright 2011 M2mobi B.V., Amsterdam, The Netherlands
+ * SPDX-FileCopyrightText: Copyright 2022 Move Agency Group B.V., Zwolle, The Netherlands
+ * SPDX-License-Identifier: MIT
+ */
+
+namespace Lunr\Core\Tests;
+
+/**
+ * This tests loading configuration files via the Configuration class.
+ *
+ * @covers \Lunr\Core\Configuration
+ */
+class ConfigurationLoadEnvironmentTest extends ConfigurationTestCase
+{
+
+    /**
+     * TestCase Constructor.
+     */
+    public function setUp(): void
+    {
+        $this->setUpArray($this->constructTestArray());
+    }
+
+    /**
+     * Test loading the environment correctly.
+     */
+    public function testLoadEnvironment(): void
+    {
+        $_ENV = [];
+
+        $_ENV['LOAD_ONE'] = 'Value';
+        $_ENV['LOAD_TWO'] = 'String';
+
+        $this->class->loadEnvironment();
+
+        $this->config['load_one'] = 'Value';
+        $this->config['load_two'] = 'String';
```

Comment:
That's exactly what it shouldn't be though. It should at the very least parse into `$this->config['load']['one']` and `$this->config['load']['two']`

Reply:
how about `API_KEYS_IOS`, is it supposed to map to `$config['api']['keys']['ios']`?